[2025-06-13T23:55:38.395+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_projeto.transform_data manual__2025-06-13T23:55:05.873332+00:00 [queued]>
[2025-06-13T23:55:38.406+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_projeto.transform_data manual__2025-06-13T23:55:05.873332+00:00 [queued]>
[2025-06-13T23:55:38.406+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 4
[2025-06-13T23:55:38.419+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transform_data> on 2025-06-13 23:55:05.873332+00:00
[2025-06-13T23:55:38.423+0000] {standard_task_runner.py:60} INFO - Started process 750 to run task
[2025-06-13T23:55:38.427+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'etl_projeto', 'transform_data', 'manual__2025-06-13T23:55:05.873332+00:00', '--job-id', '73', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpg6zcalgr']
[2025-06-13T23:55:38.429+0000] {standard_task_runner.py:88} INFO - Job 73: Subtask transform_data
[2025-06-13T23:55:38.492+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_projeto.transform_data manual__2025-06-13T23:55:05.873332+00:00 [running]> on host a984848cc8d0
[2025-06-13T23:55:38.599+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='dados@maxinutri.com.br' AIRFLOW_CTX_DAG_OWNER='kayo' AIRFLOW_CTX_DAG_ID='etl_projeto' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-06-13T23:55:05.873332+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-13T23:55:05.873332+00:00'
[2025-06-13T23:55:38.666+0000] {logging_mixin.py:188} INFO - Arquivo tratado salvo em data/dados_api_tratado.csv
[2025-06-13T23:55:38.667+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-13T23:55:38.682+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_projeto, task_id=transform_data, execution_date=20250613T235505, start_date=20250613T235538, end_date=20250613T235538
[2025-06-13T23:55:38.720+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-13T23:55:38.746+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
